project/main.py

```diff
@@ -109,7 +104,7 @@ def evaluation(experiment: Experiment) -> tuple[str, float | None, dict]:
         # We want to report the training error.
         metrics = {
             **experiment.trainer.logged_metrics,
-            **experiment.trainer.callback_metrics,
+            **experiment.trainer._metrics,
```

Comment:
Why this change again? What's the difference between `_metrics` and `callback_metrics` ?

Reply:
fixed in 2b138f6c550346ac96d62ed316c2aa6c820fa251